internal/test/e2e/run.go

```diff
@@ -217,6 +225,7 @@ func (e *E2ESession) runTests(regex string) (testCommandResult *testCommandResul
 		e.logger.V(4),
 		e.instanceId,
 		command,
+		defaultSSMTimeout,
```

Comment:
maybe we can do 
```suggestion
		e2eTimeout + e2eSSMTimeoutPadding,
```

and having that as a const (whichever value you decide)
```go
e2eSSMTimeoutPadding = 10 * Minute
```